src/main/java/uk/gov/ons/ctp/response/casesvc/domain/repository/CaseRepository.java

```diff
@@ -85,4 +87,15 @@
    * @return the cases associated with the partyId
    */
   List<Case> findByPartyId(UUID partyId);
+
+  /**
+   * Find case by its sample unit id
+   *
+   * @param sampleUnitId
+   * @return
+   */
+  @Query(
+      value = "select caze.* from casesvc.case caze where sampleunit_id = :sampleUnitId",
+      nativeQuery = true)
+  Case findBySampleUnitId(@Param("sampleUnitId") UUID sampleUnitId);
```

Comment:
This probably doesn't need to be a native query and we could possibly use the in clause https://docs.spring.io/spring-data/jpa/docs/1.5.0.RC1/reference/html/repository-query-keywords.html